app/views/layouts/application.html.erb

```diff
@@ -1,14 +0,0 @@
-<!DOCTYPE html>
```

Comment:
this needs to be put back - this is the master layout for every single page in the entire website